---
layout: software
title: "MetaCHIP"
author: Weizhi Song
comments: false
---

___


The development and application of metagenomics have provided an opportunity to study horizontal gene transfers (HGT) on the level of microbial communities. However, current HGT detection methods cannot be applied to community-level dataset or require reference genomes. Here, we present MetaCHIP, a pipeline for reference-independent and community-level HGT identification. Assessment of MetaCHIP’s performance on simulated dataset showed that it can predict HGTs having various degrees of genetic variation. The results also showed that the detection of very recent gene transfers (i.e. low level of genetic variation) from metagenomics datasets is largely affected by the reads assemble step. Assessment of MetaCHIP’s performance on real datasets confirmed the role of HGT in the spread of genes related to antibiotic resistance, in particular in the human gut microbiome. MetaCHIP provides an opportunity to study HGTs among members of microbial communities and should therefore have wide applications in the field of microbial ecology and evolution

___

**Link:**

[https://github.com/songweizhi/MetaCHIP](https://github.com/songweizhi/MetaCHIP)

___

**Publication:**

**Song WZ**, Wemheuer B, Zhang S, Steensen K, Thomas T (2018) MetaCHIP: community-level horizontal gene transfer identification through the combination of best-match and explicit phylogenetic tree approaches. **Microbiome**, (Under review).

___
